### ISSUE TITLE
No Inspired Taming for an Alpha Animals' Feralisk Clutch Mother?

### ISSUE THREAD

**Issue by TurtleShroom** [okradonkey/TrulyInspiredTaming#1]
Good evening.

I have already reported this issue there, so now I am reporting it here. With this Mod, I can tame any animal, including custom ones, but for some reason, I can't tame the Feralisk Clutch Mother. You need a Class Twenty Tamer, but this Mod ignores that. I can't right-click on the Feralisk and there's no option to tame her.

Why?

Thank you for any assistance you may render.
-TURTLESHROOM

**Comment by juanosarg** [comment]
Yeah, this is intended. No taming of clutchmothers, Fungal Husks, Mimes or Moribund Gallatross
